README.md

```diff
@@ -66,23 +66,23 @@ GNU General Public License for more details](https://www.gnu.org/licenses/gpl-2.
 As I'm a fervent [Linux](https://www.kernel.org/) user, this version of the script has only been used and tested on Linux [Debian](https://www.debian.org/). I will not be able to maintain directly the [Windows](http://windows.microsoft.com) compatibility but I'm open to pull requests on this side, as we're here to share.
 
 #### Python version
-This version of TraktForVLC has been tested using Python 2.7.9 and Python 3.4.2 It should however be compatible with the following versions:
+This version of TraktForVLC has been tested using Python 2.7.9 and Python 3.4.2. It should, however, be compatible with the following versions:
 + Python 2.6
 + Python 2.7
 + Python 3.3
 + Python 3.4
 
 #### VLC version
-This version of TraktForVLC has been tested using VLC 2.2.0 Weatherwax
+This version of TraktForVLC has been tested using VLC 2.2.0 Weatherwax.
```

Comment:
The VLC version can be updated too: VLC 2.2.6 Umbrella

Reply:
Will do!